auth-api/src/main/java/io/stargate/auth/api/resources/AuthResource.java

```diff
@@ -52,8 +53,12 @@
 
   private static final Logger logger = LoggerFactory.getLogger(AuthResource.class);
 
+  private static final int tokenMaxAge =
```

Comment:
lil nit: I think this is in seconds, could you put a comment to that effect? (not a big deal)